core/src/main/java/org/apache/calcite/rex/RexSimplify.java

```diff
@@ -2284,6 +2291,12 @@ static Predicate of(RexNode t) {
       }
       return IsPredicate.of(t);
     }
+
+    /** Returns whether this predicate can while simplifying other OR
+     * operands. */
+    default boolean allowedInOr(RelOptPredicateList predicates) {
+      return true;
```

Comment:
It seems to lack some words between `can` and `while`.

Reply:
good catch. i'll fix.